app/src/main/AndroidManifest.xml

```diff
@@ -128,6 +128,22 @@
       </intent-filter>
     </activity>
 
+    <activity
+      android:name=".signature.SignatureActivity"
+      android:screenOrientation="portrait">
+      <intent-filter>
+        <action android:name="android.intent.action.VIEW" />
+
+        <category android:name="android.intent.category.BROWSABLE" />
+        <category android:name="android.intent.category.DEFAULT" />
+
+        <data
+          android:host="app.simple.org"
+          android:pathPrefix="/hidden/teleconsult/signature"
```

Comment:
Have we confirmed this deep link schema or is it just temporary?

Reply:
Vinay gave this as a temporary link I suppose.

Reply:
This is temporary and is only meant to be used for testing this screen. In reality, a different deeplink will open another screen which will then lead to this screen.